imap/jmap_sieve.c

```diff
@@ -2019,6 +2024,26 @@ static int jmap_sieve_test(struct jmap_req *req)
         envelope = NULL;
     }
 
+    /* variables */
+    if (JNOTNULL(variables) && json_is_object(variables)) {
+        jmap_parser_push(&parser, "variables");
+
+        construct_hash_table(&vars, 64, 0);
```

Comment:
Probably I should tune this down to 16 or something?

Reply:
I wouldn't argue with 16 or 32

Reply:
Yeah 16 buckets seems like more than enough